frontend/src/api/queryBuilder/getValuesAutoComplete.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiV3Instance } from 'api';
+import { ErrorResponseHandler } from 'api/ErrorResponseHandler';
+import { AxiosError } from 'axios';
+import { ErrorResponse, SuccessResponse } from 'types/api';
+import { PayloadProps } from 'types/api/queryBuilder/getValuesAutoComplete';
+
+const getValuesAutoComplete = async (
+	attributeKey: string,
+	searchText: string,
+	aggregateOperator: undefined | string = 'sum',
+	dataSource: undefined | string = 'metrics',
+	aggregateAttribute: undefined | string = 'signoz_calls_total',
+): Promise<SuccessResponse<PayloadProps> | ErrorResponse> => {
+	try {
+		const data = await ApiV3Instance.get(
+			`/autocomplete/attribute_values?aggregateOperator=${aggregateOperator}&dataSource=${dataSource}&aggregateAttribute=${aggregateAttribute}&attributeKey=${attributeKey}&searchText=${searchText}`,
```

Comment:
this is not scalable 

let's make a utils do this job
 
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/set#return_value